style.css

```diff
@@ -155,9 +215,39 @@ footer {
     margin-right: 1rem;
   }
 
+  #pizzaTable{
+    margin: 15px;
+  }
+  
   #gallery {
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(33%, 1fr));
     background-color: white;
   }
 }
+
+/* === Home Button === */
+
```

Comment:
@JohanBester Is this the home button in the bottom right of the page that is not included in the curriculum?

Reply:
Button removed  :)